OKTA-393082 Reorganize Implement Auth flow guides (formerly Reassess sign-in docs) (#2290)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/index.md

---

> title: Implement Authorization by grant type

Reviewer: ```suggestion
title: Implement authorization by grant type
```

> excerpt: How to implement Authorization flows by grant type with Okta

Reviewer: ```suggestion
excerpt: How to implement authorization flows by grant type with Okta
```

layout: Guides
---

> This document guides you through implementing OAuth 2.0 authorization flow for your application by grant types with Okta.

Reviewer: ```suggestion
This document guides you through implementing an OAuth 2.0 authorization flow for your application by grant type with Okta.
```

Select the authorization grant-type flow to implement: <StackSelector />

## Overview

<StackSnippet snippet="overview" />

## Grant-type flow

<StackSnippet snippet="flow-diagram"/>

## Set up your app

Before you can implement authorization, you need to register your app in Okta by creating an app integration from the Admin Console.

1. In the Admin Console, go to **Applications** > **Applications**.
2. Click **Create App Integration**.
3. Select **<StackSnippet snippet="sign-in-method" inline />** as the **Sign-in method**.

<StackSnippet snippet="setup-app" />

## Install an Okta SDK

> Okta provides an array of language and framework SDKs to quickly implement authorization with Okta. See [Languages & SDKs overivew](/code/) for a list of SDKs you can download to start using with your app.

Reviewer: ```suggestion
Okta provides an array of language and framework SDKs to quickly implement authorization with Okta. See [Languages & SDKs overivew](/code/) for a list of SDKs that you can download to start using with your app.
```
